packages/core/src/transport/transport.ts

```diff
@@ -29,7 +51,41 @@ export class HttpRequest {
         reportBeaconError(e)
       }
     }
+
+    const transportIntrospection = (event: ProgressEvent) => {
+      const req = event?.currentTarget as XMLHttpRequest
+      if (req.status >= 200 && req.status < 300) {
+        return
+      }
+      const xhrTransportContext: TransportIssueContext = {
+        on_line: navigator.onLine,
+        size,
+        url,
+        try_beacon: tryBeacon,
+        bytes_limit: this.bytesLimit,
```

Comment:
FMU this value is a constant so maybe logging `size > this.bytesLimit` could be more helpful

Reply:
Removed now since `try_beacon` already has `size > this.bytesLimit`.